Directory.Build.props

```diff
@@ -339,7 +338,11 @@
     <NETCoreAppTestHostFxrPath>$([MSBuild]::NormalizeDirectory('$(TestHostRootPath)', 'host', 'fxr', '$(ProductVersion)'))</NETCoreAppTestHostFxrPath>
     <NETCoreAppTestSharedFrameworkPath>$([MSBuild]::NormalizeDirectory('$(TestHostRootPath)', 'shared', 'Microsoft.NETCore.App', '$(ProductVersion)'))</NETCoreAppTestSharedFrameworkPath>
     <!-- For UAP, we'll produce the layout and hard-link this into each test's directory -->
-    <UAPTestSharedFrameworkPath>$(TestHostRootPath)UAPLayout</UAPTestSharedFrameworkPath>
+    <UAPTestSharedFrameworkPath>$([MSBuild]::NormalizeDirectory('$(TestHostRootPath)', 'UAPLayout'))</UAPTestSharedFrameworkPath>
+
+    <RuntimePath Condition="'$(_bc_TargetGroup)' == 'netcoreapp'">$(NETCoreAppTestSharedFrameworkPath)</RuntimePath>
```

Comment:
Do you ever use this path during the allconfigurations build?  It will have the "wrong" value.  I guess we don't use ReferenceFromRuntime to resolve to it for src projects and we don't support AllConfigurations against tests...  Just do a scrub: maybe even consider changing the name of the property to avoid accidental usage (EG: TestRuntimePath or something).

Reply:
I agree, I was thinking the same but forgot to change it ❌